app/assets/stylesheets/sidebar_components.scss

```diff
@@ -4,3 +4,10 @@
   }
   border-top: 2px solid $govuk-brand-colour;
 }
+
+.sticky {
```

Comment:
Rather than name this after what it does, maybe it would be better to name it as what it is (e.g. a sidebar). Could it even be directly applied to the `sidebar-components` class (or does that not work because that `div` is at a different level in the hierarchy)?

@davidtrussler might have a better view on this matter.

Reply:
Yes I think we should avoid a generic class name like this and also name it in the way Mark suggests above. I think that's not completely straightforward though since we're applying the style to the container of the thing we want to be sticky. We may need to rename elements a bit.